.circleci/config.yml

```diff
@@ -224,7 +224,7 @@ jobs:
           command: |
             source /opt/circleci/.nvm/nvm.sh
             pushd sdk && nvm install && npm ci && npm pack && popd
-            pushd tck && nvm install && npm install && npm ci && npm run tck && popd
+            pushd tck && npm install && npm run tck && popd
```

Comment:
Can't be `npm ci` because the `package-lock.json` is not committed. This was removed/ignored for the TCK a while back — I think because the dependency on the package is in the `package.json` and these end up conflicting with cached sha hashes each time it gets rebuilt locally.